test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -280,4 +296,10 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.verification_value, transcript)
     assert_scrubbed(@gateway.options[:access_token], transcript)
   end
+
+  private
+
+  def format_dollars_amount(amount)
+    sprintf('%.2f', amount.to_f / 100).to_f
```

Comment:
Why is this method needed? Would it perhaps be simpler to just directly define values for `amount` and `tax_amount`?

Reply:
I grabbed this from the `amount` function in gateway.rb to be consistent on money formatting: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L250-L268

It looks like it is used to truncate the decimal portion of the amount, but since we are supplying the values here rather than a user, I agree that it is not needed.